schema/theme.schema.json

```diff
@@ -153,21 +282,73 @@
         "btn-color-inverted-hover": {
           "type": "string",
           "title": "Button colour - inverted hover",
-          "description": "Defines the button text/icon colour when hovered over. Should be a colour that provides good contrast against the button hover colour",
+          "description": "Defines the button text / icon colour when hovered over. Should be a colour that provides good contrast against the button hover colour.",
+          "default": "",
+          "_backboneForms": "ColourPicker"
+        },
+        "btn-color-focus": {
+          "type": "string",
+          "title": "Button colour - focus",
+          "default": "",
+          "_backboneForms": "ColourPicker"
+        },
+        "btn-color-inverted-focus": {
+          "type": "string",
+          "title": "Button colour - inverted focus",
+          "description": "Defines the button text / icon colour when focused. Should be a colour that provides good contrast against the button focus colour.",
+          "default": "",
+          "_backboneForms": "ColourPicker"
+        },
+        "btn-color-selected": {
+          "type": "string",
+          "title": "Button colour - selected",
+          "default": "",
+          "_backboneForms": "ColourPicker"
+        },
+        "btn-color-inverted-selected": {
+          "type": "string",
+          "title": "Button colour - inverted selected",
+          "description": "Defines the button text / icon colour when selected. Should be a colour that provides good contrast against the button selected colour.",
+          "default": "",
+          "_backboneForms": "ColourPicker"
+        },
+        "btn-color-locked": {
+          "type": "string",
+          "title": "Button colour - locked",
+          "default": "",
+          "_backboneForms": "ColourPicker"
+        },
+        "btn-color-inverted-locked": {
+          "type": "string",
+          "title": "Button colour - inverted locked",
+          "description": "Defines the button text / icon colour when locked. Should be a colour that provides good contrast against the button locked colour.",
+          "default": "",
+          "_backboneForms": "ColourPicker"
+        },
+        "disabled": {
+          "type": "string",
+          "title": "Disabled colour",
```

Comment:
Update button title to _Button colour - disabled_

Reply:
This should now be resolved